app/controllers/training_and_support_controller.rb

```diff
@@ -9,5 +9,7 @@ def show
     if @provider.blank?
       raise Pundit::NotAuthorizedError, "This page is only available to Early Career Teachers"
     end
+
+    @provider_name_key = @provider.name.downcase.gsub(/ /, "_")
```

Comment:
The error is a bit wrong - maybe change `Early Career Teachers` to `CIP participants`